ntoskrnl/se/srm.c

```diff
@@ -297,8 +298,7 @@ SepAdtInitializeBounds(VOID)
         (ListBounds.MinLength < 16) ||
         (ListBounds.MaxLength - ListBounds.MinLength < 16))
     {
-        DPRINT1("ListBounds are invalid: %u, %u\n",
-                ListBounds.MinLength, ListBounds.MaxLength);
+        DPRINT1("ListBounds invalid: %lu, %lu\n", ListBounds.MinLength, ListBounds.MaxLength);
```

Comment:
```suggestion
        DPRINT1("ListBounds invalid: %lu, %lu\n",
                ListBounds.MinLength, ListBounds.MaxLength);
```
can be kept in its own line.